contracts/core-token-vesting/src/contract.rs

```diff
@@ -80,9 +84,179 @@ pub fn execute(
         ExecuteMsg::Claim { denoms, recipient } => {
             claim(deps, env, info, denoms, recipient)
         }
+        ExecuteMsg::CreateCampaign {
```

Comment:
Since we only create a campaign once, and the contract is pretty much useless without a campaign being created, I would create the campaign in the instantiate function and move all the parameters into `InstantiateMsg`.

Reply:
We can still register vesting contracts directly without needing to create a campaign